src/components/Details/TranscriptSearch/TranscriptSearch.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { ChangeEventHandler, FC, useState } from "react";
+import styled from "@emotion/styled";
+
+import TranscriptItems from "./TranscriptItems";
+import { Sentence } from "../../Shared/Types/Transcript";
+
+import { fontSizes } from "../../../styles/fonts";
+import { screenWidths } from "../../../styles/mediaBreakpoints";
+import isSubstring from "../../../utils/isSubstring";
+
+const Container = styled.div({
+  display: "flex",
+  flexDirection: "column",
+  height: "100%",
+});
+
+const TitleContainer = styled.div({
+  display: "flex",
+  justifyContent: "space-between",
+  alignItems: "center",
+  marginBottom: "16px",
+  "div:first-of-type": {
+    fontSize: fontSizes.font_size_7,
+    fontWeight: 600,
+  },
+});
+
+const TranscriptContainer = styled.div({
+  minHeight: "50vh",
+  [`@media (min-aspect-ratio:5/4), (min-width:${screenWidths.desktop})`]: {
+    flex: "1 1 auto",
+    minHeight: "0",
+  },
+});
+
+interface TranscriptSearchProps {
+  searchQuery: string;
+  sentences: Sentence[];
+  jumpToVideoClip(startTime: number): void;
+  jumpToTranscript(index: number): void;
+}
+
+/**Transcript search. Note: On deskop, the parent of this component should have enough height
+ * to accomodatea a window view of the list of sentences.
+ */
+const TranscriptSearch: FC<TranscriptSearchProps> = ({
+  searchQuery,
+  sentences,
+  jumpToVideoClip,
+  jumpToTranscript,
+}: TranscriptSearchProps) => {
+  const [searchTerm, setSearchTerm] = useState<string>(searchQuery);
+  const onSearchChange: ChangeEventHandler<HTMLInputElement> = (event) =>
+    setSearchTerm(event.target.value);
+
+  const visibleSentences = sentences.filter(({ text }) => isSubstring(text, searchTerm));
+
+  return (
+    <Container>
+      <TitleContainer>
+        <div>Search transcript</div>
+        {searchTerm && <div>{visibleSentences.length} mention(s)</div>}
```

Comment:
I like mentions but I also like "reference(s)" maybe.

Reply:
Since we are using the "search" UI pattern, let's go with "result(s)" that is already established? I just used mention(s) because that's in the wireframe.

Reply:
Yep. Totally fine.